2-mandatory/3-financial-times.js

```diff
@@ -5,33 +5,58 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
 function potentialHeadlines(allArticleTitles) {
-    // TODO
+    const out = [];
+    for (z in allArticleTitles) {
+        if (ARTICLE_TITLES[z].length <= 65) {
+            out[z] = ARTICLE_TITLES[z];
```

Comment:
z is not an index, it is string. So here we can check the length of string and push it in array 'out'